content/blog/access_raspberry_pi_remotely_to_control_iot_devices.md

```diff
@@ -270,6 +274,14 @@ This command launches the HTOP interface, providing you with real-time insights
 
 By incorporating HTOP into your remote management toolkit, you can proactively address performance concerns, ensuring a smooth and reliable experience for your Raspberry Pi and IoT devices.
 
+### Monitoring Raspberry Pi's System Metrics
+
+To ensure optimal performance and prevent bottlenecks on your Raspberry Pi, monitoring its CPU, memory, and disk usage is crucial. Tools like `htop` and `psutil` provide real-time insights into resource utilization, helping you optimize resource allocation and diagnose potential issues. By remotely tracking these metrics, you can maintain better control over your Raspberry Pi, even if it is located far away.
```

Comment:
Write that one can start an a tunnel, ssh to the raspberry pi, then run htop.

Another approach is by rinning a monitoring app, and access it via the API. The detailed guied is in the link